code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,423 @@
+	desc = "A beautiful artwork depicting a fair lady and HIM, HE WEEPS, I WILL SEE HIM AGAIN. Destroyable with wirecutters."
+	icon = 'icons/obj/signs.dmi'
+	icon_state = "eldritch_painting_weeping"
+	custom_materials = list(/datum/material/wood =SHEET_MATERIAL_AMOUNT)
+	resistance_flags = FLAMMABLE
+	buildable_sign = FALSE
+	// The list of canvas types accepted by this frame, set to zero here
+	accepted_canvas_types = list()
+	// A basic proximity sensor
+	var/datum/proximity_monitor/painting_proximity_sensor
+	// For changing sensor types on subtypes
+	var/sensor_type = /datum/proximity_monitor/advanced/eldritch_painting
+	// Set to false since we don't want this to persist
+	persistence_id = FALSE
+
+// Mood applied for ripping the painting
+// These moods are here to check hallucinations and provide easier user feedback
+/datum/mood_event/eldritch_painting
+	description = "YOU, I SHOULD NOT HAVE DONE THAT!!!"
+	mood_change = -6
+	timeout = 3 MINUTES
+
+/datum/mood_event/eldritch_painting/weeping
+	description = "HE IS HERE, AND HE WEEPS!"
+	mood_change = -3
+	timeout = 10 SECONDS
+
+/datum/mood_event/eldritch_painting/weeping_heretic
+	description = "Oh such arts! They truly inspire me!"
+	mood_change = 5
+	timeout = 5 MINUTES
+
+/datum/mood_event/eldritch_painting/weeping_withdrawl
+	description = "My mind is clear from his influence."
+	mood_change = 1
+	timeout = 5 MINUTES
+
+/obj/structure/sign/painting/eldritch/Initialize(mapload, dir, building)
+	if(sensor_type)
+		painting_proximity_sensor = new sensor_type(_host = src, range = 7, _ignore_if_not_on_turf = TRUE)
+	return ..()
+
+/obj/structure/sign/painting/eldritch/wirecutter_act(mob/living/user, obj/item/I)
+	user.add_mood_event("ripped_eldritch_painting", /datum/mood_event/eldritch_painting)
+	to_chat(user, span_notice("Laughter echoes through your mind...."))
+	QDEL_NULL(painting_proximity_sensor)
+	qdel(src)
+
+/obj/structure/sign/painting/eldritch/examine(mob/living/carbon/user)
+	if(IS_HERETIC(user))
+		// If they already have the positive moodlet return
+		if("heretic_eldritch_painting" in user.mob_mood.mood_events)
+			return
+		to_chat(user, span_notice("Oh, what arts! Just gazing upon it clears your mind."))
+		// Adjusts every hallucination by -300, thus removing them if we have any
+		user.adjust_hallucinations(-300 SECONDS)
+		// Adds a very good mood event to the heretic
+		user.add_mood_event("heretic_eldritch_painting", /datum/mood_event/eldritch_painting/weeping_heretic)
+	// Do they have the mood event added with the hallucination?
+	if("eldritch_weeping" in user.mob_mood.mood_events)
+		to_chat(user, span_notice("Respite, for now...."))
+		// Remove the mood event associated with the hallucinations
+		user.mob_mood.mood_events.Remove("eldritch_weeping")
+		// Add a mood event that causes the hallucinations to not trigger anymore
+		user.add_mood_event("weeping_withdrawl", /datum/mood_event/eldritch_painting/weeping_withdrawl)
+	return ..()
```

Comment:
Call parent at the start not the end, otherwise if you early return at any point (such as if you already have the mood event) you don't get any examine message